### PR TITLE
Fix broken link on Third Party SR Integrations page

### PR DESCRIPTION
**Pull request by jas-kas**

### CODE REVIEW

**Comment by bruno-garcia** [comment]
I'll check vercel before merging this time 🙈

**Comment by bruno-garcia** [comment]
Still not loading right, not sure why: https://sentry-docs-git-jas-kas-broken-link.sentry.dev/product/integrations/session-replay/

**Comment by bruno-garcia** [comment]
This one works: https://github.com/getsentry/sentry-docs/blob/2d8677d48574a5559821f7ee4b389852a7ea1b32/src/includes/only-error-issues-note.mdx?plain=1#L3

File: src/docs/product/integrations/session-replay/index.mdx [hunk 1]
```diff
@@ -4,12 +4,14 @@ sidebar_order: 70
 description: "Learn more about third-party session replay integrations."
 ---
 
+Learn more about Sentry's session replay integrations:
+
 <Note>
-  Looking for Sentry's built-in Session Replay product? [Click here](/product/session-replay/) to learn more.
+  
+  Looking for Sentry's built-in Session Replay product? Click [here](/product/session-replay/) to learn more.
```
Comment (bruno-garcia) [inline src/docs/product/integrations/session-replay/index.mdx:11]:
```suggestion
Looking for Sentry's built-in Session Replay product? Click [here](/product/session-replay/) to learn more.
```

**Comment by bruno-garcia** [comment]
Second random attempt to get the link to render Can't see any difference between what we did here and other Note's with links on the docs

File: src/docs/product/integrations/session-replay/index.mdx [hunk 2]
```diff
@@ -4,12 +4,14 @@ sidebar_order: 70
 description: "Learn more about third-party session replay integrations."
 ---
 
+Learn more about Sentry's session replay integrations:
+
 <Note>
-  Looking for Sentry's built-in Session Replay product? [Click here](/product/session-replay/) to learn more.
+  
+Looking for Sentry's built-in Session Replay product? Click [here](/product/session-replay/) to learn more.
+
```
Comment (lizokm) [inline src/docs/product/integrations/session-replay/index.mdx:12]:
```suggestion
Looking for Sentry's built-in Session Replay product? Click <a class href="/product/session-replay/">here</a> to learn more.
  
```

**Review by lizokm — APPROVED** [review]
This seems to work. Will have to figure out what's causing the problem later since this seems like a band aid fix.

**Pull request merged**